### PR TITLE
Cucumber for admin dashboard for universities

### PR DESCRIPTION
**Pull request by Avanti0532**
Admin Dashboard is customized for universities to allow admin to add, update and delete universities. Also, cucumber for customized admin dashboard for universities is added.

## Issue Related
#91 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Code coverage
94.94 %

## Screenshot (if any change in front end)
admin

### CODE REVIEW

File: features/AdminDashboardUniversity.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  Scenario: Admin can add universities in the database
+    When I log in as an admin
+    Then I can add an university
+
+  Scenario: Admin can edit any university
+    When I log in as an admin
+    Then I can edit any university from the university database
+
+  Scenario: Admin can delete any question
+    When I log in as an admin
```
Comment (linhpha) [inline features/AdminDashboardUniversity.feature]: Typo: question -> university

File: features/AdminDashboardUniversity.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      | Linh        | Pham        | linh@gmail.com    | 12345689    |  12345689            |
+      | Jordan      | Peterson    | jordan@gmail.com  | 23456789    | 23456789         |
+
+  Scenario: Admin can see all universities in the database
+    When I log in as an admin
+    Then I can see all universities in the database
+
+  Scenario: Admin can add universities in the database
+    When I log in as an admin
+    Then I can add an university
```
Comment (hpitawela) [inline features/AdminDashboardUniversity.feature]: Typo: an => a

Comment (Avanti0532) [reply features/AdminDashboardUniversity.feature]: Fixed.

File: features/AdminDashboardUniversity.feature (hunk 1 above)
Comment (Avanti0532) [reply features/AdminDashboardUniversity.feature]: Fixed.

**Pull request merged**